doc/nrf/known_issues.rst

```diff
@@ -1365,6 +1365,18 @@ NCSDK-16644: :ref:`caf_sensor_manager` macro incorrectly converts float to senso
 
   **Workaround:** Manually cherry-pick and apply the commit with the fix from the ``main`` branch (commit hash: ``7e8c23a6632632f0cee885abe955e37a6942911d``).
 
+.. rst-class:: v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
```

Comment:
No. It would not. After the change how subscribers are processed they all goes into single big sections and there some special, zero size, tags are used to mark begin and end of the specific event subscribers. This tags are placed here thanks to name sorting and this is sensitive to the right name usage.

In old implementation every event listeners group, on every level, goes into its specyfic section. So the section named super_event_prioX is not the same like super_event_prioX_prioX. It means that in old implementation the issue should not appear.

Reply:
I missed the fact that section name used to contain the `prioX`. Please disregard the comment.